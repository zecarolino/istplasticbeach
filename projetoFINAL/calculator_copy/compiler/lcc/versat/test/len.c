int main() {
	char *s = "hello!\n";
	int i = 0;
	while (*s++) i++;
	return i;
}
