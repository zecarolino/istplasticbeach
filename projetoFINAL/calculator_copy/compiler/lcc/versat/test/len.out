VCD info: dumpfile xtop.vcd opened for output.
7
Catched Trap at time  598 clock cycles:
